session/manager.go

```diff
@@ -107,13 +113,29 @@ func (manager *Manager) Create(consumerID identity.Identity, proposalID int, con
 	}
 	sessionInstance.ConsumerID = consumerID
 	sessionInstance.Done = make(chan struct{})
-	sessionInstance.Config = config
 
-	err = manager.promiseProcessor.Start(manager.currentProposal)
+	balanceTracker, err := manager.balanceTrackerFactory(consumerID, identity.FromAddress(manager.currentProposal.ProviderID), issuerID)
 	if err != nil {
 		return
 	}
 
+	// stop the balance tracker once the session is finished
+	go func() {
+		<-sessionInstance.Done
+		balanceTracker.Stop()
+	}()
+
+	go func() {
+		err := balanceTracker.Start()
+		if err != nil {
+			log.Error(managerLogPrefix, "balance tracker error: ", err)
+			destroyErr := manager.Destroy(consumerID, string(sessionInstance.ID))
```

Comment:
We convert sessionID to string here, then in Destroy method we convert it back :), maybe refactor Destroy method..

Reply:
I'll create a separate task for the destroy then, since it's used in a couple of places.

Reply:
https://github.com/mysteriumnetwork/node/issues/705